lua/entities/sent_spawnpoint/init.lua

```diff
@@ -29,64 +34,57 @@ function unlinkPlayerFromSpawnPoint( player, spawnPoint )
 end
 
 function unlinkAllPlayersFromSpawnPoint( spawnPoint, excludedPlayers )
-    if ( IsValid(spawnPoint) ) then
-        local linkedPlayers = spawnPoint.LinkedPlayers
-        local spawnPointOwner = spawnPoint:CPPIGetOwner()
-
-        for player, _ in pairs( linkedPlayers ) do
-            local playerIsExcluded = excludedPlayers[player]
-            local playerOwnsSpawnPoint = spawnPointOwner == player
-			
-            if ( not playerIsExcluded and not playerOwnsSpawnPoint ) then
-                unlinkPlayerFromSpawnPoint( player, spawnPoint )
-                player:PrintMessage(4, "You've been unlinked from a Spawn Point!")
-            end
+    if not IsValid( spawnPoint ) then return end
+
+    local linkedPlayers = spawnPoint.LinkedPlayers
+    local spawnPointOwner = spawnPoint:CPPIGetOwner()
+    
+    for player, _ in pairs( linkedPlayers ) do
+        if not ( excludedPlayers[player] or spawnPointOwner == player ) then
```

Comment:
Can you assign `excludedPlayers[player] or spawnPointOwner == player` to a variable in here which describes what you're checking?